script.js

```diff
@@ -1,76 +1,92 @@
-const input = document.getElementById("input");
-const button = document.getElementById("button");
+const EventKey = {
+  ENTER: 'enter'
+};
 
-document.addEventListener("DOMContentLoaded", function () {
-  button.addEventListener("click", callback);
-  input.addEventListener("keypress", function (event) {
-    if (event.key === "Enter") {
-      button.click();
-    }
-  });
-  input.addEventListener("change", () => {
-    chrome.storage.sync.set({ input: input.value });
+const ColorList = [
+  "hotpink",
+  "deepPink",
+  "MediumVioletRed",
+  "PaleVioletRed",
+  "Coral",
+  "Tomato",
+  "Orange",
+  "Chocolate",
+  "Peru",
+  "Sienna",
+  "SaddleBrown",
+  "Maroon",
+  "Crimson",
+];
+
+const initializeInputs = () => {
+  const inputElement = document.getElementById("input");
+  const buttonElement = document.getElementById("button");
+
+  buttonElement.addEventListener("click", () => handleFormSubmit(inputElement));
```

Comment:
These are not initializations, this could be moved to different method like attachEventListeners();
inputElement and buttonElement could be moved out so attachEventListeners could have access to it and only initialize them there.